compliance_checker/tests/test_cf.py

```diff
@@ -2957,6 +2957,27 @@ def test_polygon_geometry(self):
             "which defines the interior_ring attribute, "
             "the part_node_count attribute must also be present" in results[0].msgs
         )
+        dataset.close()
+        # TEST CONFORMANCE 7.5 REQUIRED 20/20
+        # part_node_count and interior_ring variables should have the same dimensions
+        # need diskless, no persist to make modifications to file without writing to file
+        # to file upon close
+        dataset = Dataset(
```

Comment:
@ocefpaf , is there a simpler or more elegant way to modify an existing NetCDF file's layout without persisting to disk?